1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_1_Password_Validator/src/index.spec.ts

```diff
@@ -1,5 +1,79 @@
+import { PasswordValidator } from './index';
 
```

Comment:
Incredible use of the table to complete this one! I like this $not pattern you're using. We might have to share this one as a guide. Nicely done!